src/ch/blinkenlights/android/vanilla/LibraryPagerAdapter.java

```diff
@@ -471,6 +466,7 @@ public void clearLimiter(int type)
 				mPendingFileLimiter = null;
 			} else {
 				mFilesAdapter.setLimiter(null);
+				loadSortOrder(mFilesAdapter);
```

Comment:
Hmm.. looking at the existing code, i don't see a reason why loadSortOdrer() was called for any other adapter as the sort order never depends on any limiter settings and stays valid for the lifetime of the adapter.

I'll remove the loadSortOrder() calls for the other adapter types after the merge